c#/Mandoline.Api.Examples/Client.Gui/AppSettings.config.example

```diff
@@ -1,11 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Is this not a bit redundant as it's the same file as the AppSettings.config in the same directory? Perhaps it may be a bit cleaner to add the example to the docs? Let me know what you think. If you think it's better I will leave it there.

Reply:
Actually my mistake, I see that I have the AppSettings.config as part of my git ignore. I will push the example files now.